slasher/db/attester_slashings.go

```diff
@@ -3,13 +3,17 @@ package db
 import (
 	"bytes"
 
+	"github.com/prysmaticlabs/prysm/shared/bytesutil"
+
 	"github.com/boltdb/bolt"
 	"github.com/gogo/protobuf/proto"
 	"github.com/pkg/errors"
 	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
 	"github.com/prysmaticlabs/prysm/shared/hashutil"
 )
 
+const latestEpochKey = "LATEST_EPOCH_DETECTED"
```

Comment:
Can this live in some slasher/db/schema.go file where we define buckets and other globals of the slasher db?